lib/startErrorHandling.js

```diff
@@ -26,13 +26,28 @@ var startErrorHandling = function startErrorHandling(store) {
 
   raven.config(_constants.CONFIG.ravenURL).install();
 
+  var checkUUID = function checkUUID(_ref, _ref2) {
+    var prevWorker = _ref.worker;
+    var curWorker = _ref2.worker;
+
+    if (!prevWorker.get('uuid') && curWorker.get('uuid')) {
```

Comment:
so the user context gets set elsewhere too? just wondering why we're not always setting this in the user context